app-admin/qtpass/qtpass-1.3.2.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PLOCALES="ar_MA ca cs_CZ de_DE de_LU el_GR en_GB en_US es_ES fr_BE fr_FR fr_LU
```

Comment:
We don't use `l10n` anymore unless there are separate tarballs for translations.

Reply:
Removed.